api/service.go

```diff
@@ -51,11 +51,13 @@ func (s *fixedService) Login(username, password string) (users.User, error) {
 
 }
 
-func (s *fixedService) Register(username, password, email string) (string, error) {
+func (s *fixedService) Register(username, password, email, first, last string) (string, error) {
```

Comment:
Wouldn't it be cleaner to have the constructor receive all these arguments and return a new user with all the fields properly assigned, including the calculation of the pass hash and all that? I'm thinking in an MVC way, where the model is fat and has all this logic in it, but that mindset might not be fit for this scenario.
What do you think?